src/tests/utilsTest.ts

```diff
@@ -36,7 +37,9 @@ import * as fs from 'fs-extra'
     }
 
     @test async "TestDoOnTargetBranch" () {
-        const cfg = await createDevOpsEnv('devops');
+        const rootName = 'test-root'
```

Comment:
Seems the style is inconsistent, some rows are not ended by `;`, does it have tslint check.

Reply:
The repo doesn't have any tslint check. I will create a separate PR to fix.